packages/runner-ct/lib/runner-ct.ts

```diff
@@ -2,7 +2,8 @@ import Debug from 'debug'
 import _ from 'lodash'
 import path from 'path'
 import send from 'send'
-import { Cfg, ProjectCt, SpecsStore } from '@packages/server-ct'
+import { Cfg, ProjectCt } from '@packages/server-ct'
+import { SpecsStore } from '@packages/server/lib/specs-store'
```

Comment:
nit: Do you think we could make this surface to `packages/server/index` so that we don't have to dig in the package?

When we use npm packages in a non standard way, it will force us to keep old outdated package managers like yarn. I would prefer paying this debt whenever we can.

Reply:
It's a bit weird - the top level `index.js` file in `server` has a specific export that I'm not sure we should change:

```js
module.exports = require('./lib/cypress').start(process.argv)
```

I definitely agree we should rethink how things are exported, though. I'm not sure changing this here is a good idea - we have a strong culture of digging into `server` to import things. We should think about this change a bit more before we do too much.

![image](https://user-images.githubusercontent.com/19196536/122701371-71b8eb80-d290-11eb-90e4-bf3b3edb441f.png)

Reply:
+1 to fixing this all-around at some point.